middleware/src/main/java/identification/ObjectId.java

```diff
@@ -0,0 +1,32 @@
+
+package identification;
+
+import java.util.UUID;
+
+/**
+ * Defines a valid (globally unique) identifier for each remote object instance, 
+ * which is a 64-bit number value.
+ * 
+ * @author Yuri Alessandro Martins 
+ * @version 1.0
+ */
+public class ObjectId {
+	
+	public UUID objectId;
+	
+	/**
+	 * Construct new ObjectId
+	 */
+	public ObjectId() {
+		this.objectId = UUID.randomUUID();
+	}
+	
```

Comment:
Create a getter for the UUID's least significant bits

Reply:
done